common/autoware_auto_perception_rviz_plugin/src/object_detection/object_polygon_detail.cpp

```diff
@@ -160,19 +160,19 @@ visualization_msgs::msg::Marker::SharedPtr get_twist_covariance_marker_ptr(
   // ellipse orientation
   marker_ptr->pose.orientation.x = 0.0;
   marker_ptr->pose.orientation.y = 0.0;
-  marker_ptr->pose.orientation.z = std::sin(phi / 2.0);
-  marker_ptr->pose.orientation.w = std::cos(phi / 2.0);
+  marker_ptr->pose.orientation.z = std::sin(phi * 0.5);
+  marker_ptr->pose.orientation.w = std::cos(phi * 0.5);
 
   // ellipse size
   marker_ptr->scale.x = sigma1 * confidence_interval_coefficient;
   marker_ptr->scale.y = sigma2 * confidence_interval_coefficient;
   marker_ptr->scale.z = 0.05;
 
-  marker_ptr->lifetime = rclcpp::Duration::from_seconds(0.5);
+  marker_ptr->lifetime = rclcpp::Duration::from_seconds(0.15);
   marker_ptr->color.a = alpha;
-  marker_ptr->color.r = 1.0;
-  marker_ptr->color.g = 0.2;
-  marker_ptr->color.b = 0.4;
+  marker_ptr->color.r = 0.2;
```

Comment:
[imo, you can ignore this] If possible, I prefer there are some comments about color description because we can't image color from rgb values :)